jdk-wrapper.sh

```diff
@@ -189,6 +189,10 @@ extension="tar.gz"
 if [ "${JDKW_PLATFORM}" = "macosx-x64" ]; then
   extension="dmg"
 fi
+# JDK 6 and before use .bin extension, note no specific macosx download available for these versions (using linux)
+if [ "${JDKW_VERSION:2:1}" -le "6" ]; then
+  extension="bin"
```

Comment:
If my understanding above is correct then you should nest this an os check for linux before doing the version check and then default to bin if both are true.

Please also add an asterisk to line 71 and describe this exception as a foot-note in the header comment and do the same in the Configuration section of the README.md file.